luminaire/model/lad_structural.py

```diff
@@ -442,12 +436,72 @@ def _training(self, data, ts_start, ts_end, min_ts_length=None, min_ts_mean=None
 
         return result, order
 
-    def train(self, data, optimize=False, **kwargs):
+    def _validate_model(self, data, hyper_params, result):
+        """
+        This function validates a newly trained model before publishing
```

Comment:
B: Can you add more details on what does it mean by validating? Like what value does it add

Reply:
Done!

Reply:
The new comments does not add much details on what it means by validating here. Not sure if that is captured anywhere else?

Reply:
I updated the documentation. Also see the comments added where the function has been called.